typescript/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@revenuecat/purchases-typescript-internal",
   "title": "Purchases typescript internal shared code",
-  "version": "6.3.1",
+  "version": "6.3.2",
```

Comment:
I believe this deployment won't have the fix so the hotfix is not deployed as the latest right? We would need to retag the latest version after deploying this

Reply:
just did this